packages/commerce-sdk-react/src/auth/storage.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {BaseStorage, ServerStorage} from './storage'
+
+const {window} = global
+const key = 'key'
+const value = 'value'
+
+const testCases = [
+    {
+        description: 'ServerStorage works without options',
```

Comment:
```suggestion
        description: 'ServerStorage works with empty options',
```
"without" to me implies `undefined`, not empty object.

Reply:
I went the other way here and set the `storageOptions` to `undefined` as that is actually what I meant to test.